strongbox-web-core/src/main/java/org/carlspring/strongbox/app/boot/DefaultBootServlet.java

```diff
@@ -0,0 +1,138 @@
+package org.carlspring.strongbox.app.boot;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+public class DefaultBootServlet
+        extends HttpServlet
+{
+
+    private static final Logger logger = LoggerFactory.getLogger(DefaultBootServlet.class);
+
+    private static final String assetsPath = "/static/assets/";
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException
+    {
+        // normalize requested path / uri
+        Path path = (new File(request.getRequestURI())).toPath().normalize();
+        String requestedUri = path.toString();
+
+        // Strip context path from the beginning.
+        if (StringUtils.isNotBlank(getServletContext().getContextPath()))
+        {
+            requestedUri = requestedUri.substring(getServletContext().getContextPath().length());
+        }
+
+        String requestedResource = "/index.html";
+
+        // Set default status code for all requests
+        response.setStatus(HttpStatus.SERVICE_UNAVAILABLE.value());
```

Comment:
why `SERVICE_UNAVAILABLE` if default `requestedResource` is `index.html` ?

Reply:
We need to respond with `SERVICE_UNAVAILABLE` to all requests, because otherwise health monitoring tools will be fooled that Strongbox is actually live when it's not.
The only exception here are the assets under `/static/assets`, because browsers complain about it and won't retrieve them as well.

Reply:
ok :)